.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,54 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: 'FEATURE REQUEST: '
+labels: ':type: enhancement'
+assignees: ''
+
+---
+
+*NOTE: This issue template is being created and may not be complete.*
+
+*NOTE: Please only submit feature requests to this repository for features
+requested for the testing repository itself. General feature requests for Rocky
+Linux should be submitted for general discussion in the [Rocky Linux
+Mattermost](https://chat.rockylinux.org) or [Rocky Linux
+Forums](https://forums.rockylinux.org).*
+
+---
+
+**Is your feature request related to a problem? Please describe.**
+
+A clear and concise description of what the problem is. For example...
+
+> It would improve my workflow if...
+
+...or...
+
+> It impedes my work when...
+
+---
+
+**Describe the solution you'd like**
```

Comment:
What are you thoughts on naming the sections as the creator of a PR would, rather than instructions? For example changing `Describe the solution you'd like` to `Proposed Solution`, and moving the instructions to the body of the section.

Reply:
Group meeting to possibly tweak this point as we understand how the automation fits in vs user interacting with it.